### PR TITLE
Add Random NPC names

### PR DESCRIPTION
**Pull request by wesbaker**
Hey there, I figured it'd be handy to have some names along with the random NPCs. I also did some tidying up while I was looking around.

### CODE REVIEW

**Comment by wesbaker** [comment]
@funkaoshi any thoughts on this?

**Comment by funkaoshi** [comment]
Damn sorry I should turn on notifications from Github again.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,108 @@
+### https://raw.github.com/github/gitignore/38b189d3b9ea5a14f7177db78d13e2e1cc0e0092/Python.gitignore
```
Comment (funkaoshi) [inline .gitignore:1]: Weird: I didn't already have a .gitignore?

**Review by funkaoshi — COMMENTED** [review]
This seems fine. I'll check it out and run it locally. I had purposefully left names out because I wanted the generated characters to be a blank slates of sorts. For big NPC lists I can see how it's handy, though.

File: .gitignore (hunk 1 above)
Comment (wesbaker) [reply .gitignore:1]: Didn't look like it and the PR didn't find it 🤷‍♂️

**Comment by wesbaker** [comment]
> I had purposefully left names out because I wanted the generated characters to be a blank slates of sorts. For big NPC lists I can see how it's handy, though.

I figured that was the case, but I figured I'd submit the PR.

**Comment by funkaoshi** [comment]
Merged! Can maybe bug people for more fantasy names on G+.

**Pull request merged**